### PR TITLE
[commands] added docs

### PR DESCRIPTION
**Pull request by akosyakov**

### CODE REVIEW

File: packages/core/src/common/command.ts [hunk 1]
```diff
@@ -9,31 +9,74 @@ import { injectable, inject, named } from "inversify";
...
 import { ContributionProvider } from './contribution-provider';
 
+/**
+ * A command is an unique identifier of a function
+ * which can be executed by a user via a keyboard shortcut,
+ * a menu action or directly.
+ */
 export interface Command {
+    /**
+     * An unique identifier of this command.
```
Comment (epatpol) [inline packages/core/src/common/command.ts]: I think it's "A unique".

File: packages/core/src/common/command.ts [hunk 2]
```diff
@@ -52,6 +95,11 @@ export class CommandRegistry implements CommandService {
         }
     }
 
+    /**
+     * Register the given command and handler if present.
+     *
+     * Throw if a command is already registerd for the given command identifier.
```
Comment (marcdumais-work) [inline packages/core/src/common/command.ts]: typo: registered

File: packages/core/src/common/command.ts [hunk 3]
```diff
@@ -9,31 +9,74 @@ import { injectable, inject, named } from "inversify";
...
 
+/**
+ * A command is an unique identifier of a function
+ * which can be executed by a user via a keyboard shortcut,
+ * a menu action or directly.
+ */
 export interface Command {
+    /**
+     * An unique identifier of this command.
+     */
```
Comment (hexa00) [inline packages/core/src/common/command.ts]: s/An/A

File: packages/core/src/common/command.ts (hunk 2 above)
Comment (hexa00) [inline packages/core/src/common/command.ts]: s/registerd/registered

File: packages/core/src/common/command.ts [hunk 4]
```diff
@@ -107,6 +169,9 @@ export class CommandRegistry implements CommandService {
         return Promise.reject(`The command '${command}' cannot be executed. There are no active handlers available for the command.${argsMessage}`);
     }
 
+    /**
+     * A visible handler for the given command or `undefined`.
```
Comment (hexa00) [inline packages/core/src/common/command.ts]: Get a visible... or return undefined

File: packages/core/src/common/command.ts [hunk 5]
```diff
@@ -119,6 +184,9 @@ export class CommandRegistry implements CommandService {
         return undefined;
     }
 
+    /**
+     * An active handler for the given command or `undefined`.
+     */
```
Comment (hexa00) [inline packages/core/src/common/command.ts]: dito

File: packages/core/src/common/command.ts [hunk 6]
```diff
@@ -142,10 +213,16 @@ export class CommandRegistry implements CommandService {
         return commands;
     }
 
+    /**
+     * A command for the given command identifier.
```
Comment (hexa00) [inline packages/core/src/common/command.ts]: Get a command...

File: packages/core/src/common/command.ts [hunk 7]
```diff
@@ -142,10 +213,16 @@ export class CommandRegistry implements CommandService {
...
+     */
     getCommand(id: string): Command | undefined {
         return this._commands[id];
     }
 
+    /**
+     * Identifiers of all registered commands.
```
Comment (hexa00) [inline packages/core/src/common/command.ts]: Get all registered commands identifiers

**Pull request merged**